src/odemis/driver/test/tescan_test.py

```diff
@@ -93,6 +93,7 @@ def test_creation(self):
                 sed = child
             elif child.name == CONFIG_SCANNER["name"]:
                 scanner = child
+            # TODO all unused...
```

Comment:
Well, they are mostly there so that the next person reading the code will know which child is created. So somehow useful ;-)